src/umpire/op/CMakeLists.txt

```diff
@@ -28,8 +28,7 @@ set (umpire_op_sources
   MemoryOperation.cpp
   MemoryOperationRegistry.cpp)
 
-set (umpire_op_depends
-  umpire_util)
+set (umpire_op_depends)
```

Comment:
So is this just setting umpire_op_depends to nothing?  Should this and the reference line below be removed?

Reply:
No, because if CUDA or ROCm are used, they are added to the dependency list.